### PR TITLE
docs: v3.0.0 release notes

### PR DESCRIPTION
**Pull request by ryanchristo**
## Description

Adds release notes for `v3.0.0`.

---

### Author Checklist

*All items are required. Please add a note to the item if the item is not applicable and
please add links to any relevant follow up issues.*

I have...

- [ ] included the correct [type prefix](https://github.com/commitizen/conventional-commit-types/blob/v3.0.0/index.json) in the PR title
- [ ] added `!` to the type prefix if API or client breaking change
- [ ] targeted the correct branch (see [PR Targeting](https://github.com/cosmos/cosmos-sdk/blob/master/CONTRIBUTING.md#pr-targeting))
- [ ] provided a link to the relevant issue or specification
- [ ] followed the guidelines for [building modules](https://github.com/cosmos/cosmos-sdk/blob/master/docs/building-modules)
- [ ] included the necessary unit and integration [tests](https://github.com/cosmos/cosmos-sdk/blob/master/CONTRIBUTING.md#testing)
- [ ] added a changelog entry to `CHANGELOG.md`
- [ ] included comments for [documenting Go code](https://blog.golang.org/godoc)
- [ ] updated the relevant documentation or specification
- [ ] reviewed "Files changed" and left comments if necessary
- [ ] confirmed all CI checks have passed

### Reviewers Checklist

*All items are required. Please add a note if the item is not applicable and please add
your handle next to the items reviewed if you only reviewed selected items.*

I have...

- [ ] confirmed the correct [type prefix](https://github.com/commitizen/conventional-commit-types/blob/v3.0.0/index.json) in the PR title
- [ ] confirmed `!` in the type prefix if API or client breaking change
- [ ] confirmed all author checklist items have been addressed 
- [ ] reviewed state machine logic
- [ ] reviewed API design and naming
- [ ] reviewed documentation is accurate
- [ ] reviewed tests and test coverage
- [ ] manually tested (if applicable)

### CODE REVIEW

**Review by clevinson — APPROVED** [review]

**Comment by clevinson** [comment]
@ryanchristo can we add changelog entries as well?

File: RELEASE_NOTES.md [hunk 1]
```diff
@@ -1,11 +1,20 @@
...
+### Baskets
 
-This release (`v2.1.0`) hard codes an emergency height-based upgrade which introduces a consensus-breaking change at height `3126912` that resolves the issue. All production validators and full-nodes must update to `v2.1.0` prior to the upgrade height (estimated for Friday Nov 26, 17:00 UTC).
+Regen Ledger `v3.0.0` updates the ecocredit module to include basket functionality, enabling the aggregation of heterogeneous ecosystem service credits into baskets. Credits from different credit classes and batches that meet a defined criteria can be deposited within a basket in exchange for basket tokens. The basket tokens are fully fungible with other tokens from the same basket, and are tracked in the standard bank module, which means these assets will be made visible in wallets like Keplr, and also able to be transferred via IBC to other chains in the Cosmos ecosystem. Each basket token can later be redeemed for an underlying ecocredit from the given basket, and the ecocredits received may be retired by the account receiving them (for offsetting emissions).
```
Comment (technicallyty) [inline RELEASE_NOTES.md:7]:
```suggestion
Regen Ledger `v3.0.0` updates the ecocredit module to include basket functionality, enabling the aggregation of heterogeneous ecosystem service credits into baskets. Credits from different credit classes and batches that meet a defined criteria can be deposited within a basket in exchange for basket tokens. The basket tokens are fully fungible with other tokens from the same basket, and are tracked in the standard bank module, which means these assets will be made visible in wallets like Keplr, and also able to be transferred via IBC to other chains in the Cosmos ecosystem. Basket tokens can later be redeemed for underlying ecocredits from the given basket, and the ecocredits received may be retired by the account receiving them (for offsetting emissions).
```
I think that makes it sound like it's a 1:1 relationship, but you could have, say, 500 tokens : 1 credit ratio

File: RELEASE_NOTES.md [hunk 2]
```diff
@@ -1,11 +1,20 @@
...
+
+Regen Ledger `v3.0.0` includes a scoped-down, minimum-viable basket implementation with the intention to bring IBC compliant carbon credits to the interchain. For more information about the full specification for basket functionality, see the [basket specification](https://github.com/regen-network/regen-ledger/blob/master/rfcs/002-baskets-specification.md).
+
+The MVP version of baskets proposed in Regen Ledger `v3.0.0` differs from the full specification in the RFC above in that:
+
+- `BasketCritera` is restricted to only allow for:
```
Comment (technicallyty) [inline RELEASE_NOTES.md:13]: should we include credit type here?

File: RELEASE_NOTES.md (hunk 1 above)
Comment (clevinson) [reply RELEASE_NOTES.md:7]: Don't we always have a 1-1 ratio (modulo the basket exponent)?

Comment (technicallyty) [reply RELEASE_NOTES.md:7]: ah right, for non prefixed tokens. im guessing users wont see the prefixed tokens ever so ill mark this as resolved

**Review by technicallyty — APPROVED** [review]

**Pull request merged**